### PR TITLE
Fix/env creation

### PR DESCRIPTION
**Pull request by pablocast**
## Purpose
Fixed the script for automating the infra creation:
-assigned the env variables with the database credentials, necessary to insert data

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: set_up.sh [hunk 1]
```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 ### PARAMETERS ###
-prefix="demoassistant"
+prefix="suraagent"
```
Comment (lfbraz) [inline set_up.sh:3]: Maybe it is better to keep a more generalist prefix (as demoassistant, demoagent, etc.).

**Review by lfbraz — COMMENTED** [review]
I've added some comments regarding keeping the code generalist. Could you please review if they make sense?

File: set_up.sh (hunk 1 above)
Comment (pablocast) [reply set_up.sh:3]: done

File: docker-compose.yml [hunk 2]
```diff
@@ -18,4 +18,4 @@ services:
 
 secrets:
   service-account:
-    file: ./secrets/SECRET_NAME_PLACEHOLDER
+    file: ./secrets/demomeli-439613-11b04c382041.json
```
[the remark this answers is not in the document]
Comment (pablocast) [reply docker-compose.yml:21]: done

**Pull request merged**